data/ui_scripts/server_list/serverlist.lua

```diff
@@ -10,29 +10,48 @@ game:addlocalizedstring("MENU_PING", "Ping")
 game:addlocalizedstring("SERVERLIST_PLAYER_COUNT", "&&1 Players")
 game:addlocalizedstring("SERVERLIST_SERVER_COUNT", "&&1 Servers")
 
+game:addlocalizedstring("SERVERLIST_ADD_TO_FAVOURITES", "Add to favourites")
+game:addlocalizedstring("SERVERLIST_ADD_TO_BLACKLIST", "Add to blacklist")
+game:addlocalizedstring("SERVERLIST_REMOVE_FROM_FAVOURITES", "Remove from favourites")
+game:addlocalizedstring("SERVERLIST_REMOVE_FROM_BLACKLIST", "Remove from blacklist")
+game:addlocalizedstring("SERVERLIST_CLEAR_LIST", "Clear list")
+
+game:addlocalizedstring("LUI_MENU_BLACKLIST", "Blacklist")
+game:addlocalizedstring("LUI_MENU_PUBLIC", "Public")
+game:addlocalizedstring("LUI_MENU_FAVOURITES", "Favourites")
+game:addlocalizedstring("LUI_MENU_SETTINGS", "Settings")
+
+game:addlocalizedstring("LUA_MENU_SERVERTYPE", "TYPE")
```

Comment:
Uhm… I didn't find any examples where `LUI_MENU_SETTINGS` or `LUA_MENU_SERVERTYPE` are used

Reply:
Forgot to remove it, LUI_MENU_SETTINGS  was far an extra features that i was working on.